src/natcap/invest/utils.py

```diff
@@ -714,7 +613,15 @@ def read_csv_to_dataframe(
 
     Args:
         path (str): path to a CSV file
-        to_lower (bool): if True, convert all column names to lowercase
+        index_col (str): name of column to use as the dataframe index. If
+            ``cols_to_lower``, this column name and the dataframe column names
+            will be lowercased before they are compared. If ``usecols``
+            is defined, this must be included in ``usecols``.
+        usecols (list(str)): list of column names to subset from the dataframe.
+            If ``cols_to_lower``, these names and the dataframe column names
+            will be lowercased before they are compared.
```

Comment:
Since `index_col` is required to be in `usecols`, would it make sense to simply add it into `usecols` if it isn't already there?  It seems like this might be a minor bit of de-duplication when writing out a function call, but I'm curious what you think!

Reply:
That certainly could make sense! I was going for consistency with `pandas.read_csv`, which will raise an error if you don't include the value of `index_col` in your `usecols` list. I think it also parallels nicely with the structure of the `MODEL_SPEC` - when we implement #1328, we might pipe in the spec `columns`, which already include the index column.

Reply:
That all makes sense to me!